### ISSUE TITLE
Submit proposal to IAAF - Due June 16th

### ISSUE THREAD

**Issue by Maroyafaied** [DOI-ONRR/onrr.gov-site#2680]
Our teams are busy making plans for the 2023 Interagency Accessibility Forum (IAAF) and to celebrate accessibility as the foundation for the Federal Government to serve as a model where all employees are treated with dignity and respect. 

The 2023 IAAF is scheduled for November 7 -9, 2023 as an in-person event held at the Federal Deposit Insurance Corporation (FDIC) L. William Seidman Center, with some portions available virtually for individuals unable to attend in person.

If you are interested in presenting a workshop or discussion, please see the attached call for proposals for more information. Submissions are due Friday, June 16, 2023. Proposals can be submitted to iaaf@gsa.gov.

Thank you, and have an accessible day!

**Comment by lpgoldstein** [comment]
@maroyafaied let's do our inclusive culture proposal

**Comment by Maroyafaied** [comment]
@lpgoldstein we already presented the 508 project previously to IAAF but I think we can still have the topic around culture and alter the presentation a bit  .

**Comment by lpgoldstein** [comment]
@Maroyafaied Yeah I think we can do the what happened next and culture portion or we can also submit the consent form discussion that Erin is doing for gov ux. Or both.

**Comment by cthomasONRR** [comment]
@Maroyafaied We could propose the 4-point hybrid testing. Most of that content is already developed so it will be easy to submit if we can propose multiples.

**Comment by lpgoldstein** [comment]
@Maroyafaied I already started a document in our files and copied this info in. Will share.

**Comment by lpgoldstein** [comment]
@Maroyafaied @eelzi-ONRR file folder for the proposal https://doimspp.sharepoint.com/:f:/r/sites/NRRD/Shared%20Documents/General/Outreach/Proposals%20%26%20Presentations/FY23/IAAF%20Proposal%20FY24?csf=1&web=1&e=ob1oLZ

**Comment by Maroyafaied** [comment]
testing slack feed

**Comment by lpgoldstein** [comment]
@eelzi-ONRR looks great - just a few grammar edits. I feel blogs coming out of these proposals no matter what.

**Comment by Maroyafaied** [comment]
@eelzi-ONRR @lpgoldstein Great job, I made minor changes/suggestions. Thanks

**Comment by eelzi-ONRR** [comment]
@lpgoldstein @Maroyafaied I incorporated your changes and made one very minor change to Lindsay's
